hash/psx.xml

```diff
@@ -37382,6 +37397,262 @@ The entries in this section are intended to replace the existing "low-grade" Jap
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 08).bin" size="24020976" crc="6137c2e6" sha1="4bcab9780733d26f0eb7b7037f5cedc8b587ce05"/>
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 09).bin" size="23552928" crc="c1cbac5f" sha1="4339f149cd25a747940e84e5572189f6bae27fb5"/>
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 10).bin" size="32358816" crc="6bf24960" sha1="d39579c124a47a19b36639761c1581551792d1eb"/>
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 11).bin" size="22863792" crc="3dbb911b" sha1="38a994f0546473382b735dd666fbefddec4454ae"/>
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 12).bin" size="28129920" crc="525b1ddd" sha1="f9a3912ebd06a766a9430bdf69acea37af217be8"/>
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 13).bin" size="33062064" crc="2884a6c3" sha1="61cf3423a5d67360c25652abc821f6dff54a47cc"/>
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 14).bin" size="29781024" crc="5ee633f5" sha1="66ef5382f4b236c76bc6089af75c0b21865494ff"/>
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 15).bin" size="23362416" crc="4b8be2b0" sha1="757e60ea242cd81f216dc7d29826501bacde76e6"/>
+		<rom name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan) (Track 16).bin" size="33908784" crc="857edda8" sha1="28a06c34d53442f736fed283c086340d7c56321e"/>
+		-->
+		<description>Ace Combat 3: Electrosphere (Direct Audio with AppenDisc) (Japan)</description>
+		<year>1999</year>
+		<publisher>Namco</publisher>
+		<info name="developer" value="Namco"/>
+		<info name="language" value="Japanese"/>
+		<info name="serial" value="SLPM-80453"/>
+		<info name="speech" value="Japanese" />
+		<sharedfeat name="compatibility" value="NTSC-J"/>
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Ace Combat 3 - Electrosphere - Direct Audio with AppenDisc (Japan)" sha1="fb4ebde7044c90c6beabe512921e0f7d2f694604"/>
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="aconcagu" supported="no">
+		<!--
+		http://redump.org/disc/14630
+		<rom name="Aconcagua (Japan) (Disc 1).cue" size="92" crc="f601804b" sha1="069467f35825092dc5fc5e8dc2ff1d8bc3933199"/>
+		<rom name="Aconcagua (Japan) (Disc 1).bin" size="665006832" crc="c262518d" sha1="ea94e096e687c8e8a7c3c99dcdc36d99cd27d8f1"/>
+
+		http://redump.org/disc/14629
+		<rom name="Aconcagua (Japan) (Disc 2).cue" size="92" crc="f75ca69a" sha1="1265ef3a1cac866405c8a50d0780bbd954092c3f"/>
+		<rom name="Aconcagua (Japan) (Disc 2).bin" size="725476752" crc="28783402" sha1="4150b207318d7f5376e43f6301552b28cbee6169"/>
+		-->
+		<description>Aconcagua (Japan)</description>
+		<year>2000</year>
+		<publisher>Sony</publisher>
+		<notes><![CDATA[
+The game only boot on PAL
```

Comment:
This replaces the set with something that doesn't seem to work properly, which is generally due of a bootlegged source, are we sure this is a _proper_ dump?

Reply:
On no-intro site, the dump info is marked as original.
But it isn't confirmed.